deploy/scripts/create_kind_cluster.sh

```diff
@@ -35,7 +35,8 @@ echo
 echo "Download kind executable with dual stack support"
 # We need to replace kind executable and node image
 # with official release once dual stack is fully supported by upstream.
-curl -L https://github.com/song-jiang/kind/releases/download/dualstack-1.17.0/kind -o ${KIND}
+#curl -L https://github.com/song-jiang/kind/releases/download/dualstack-1.17.0/kind -o ${KIND}
```

Comment:
This change is causing the CI issue.

Reply:
Thanks @tmjd I made that change to try to run `make test` on my workstation (I had to use kind binary build for mac) and forget to revert it 🤦 Pushing a fix.

Reply:
That's what I was guessing had probably happened.